examples/get-started/pure-js/leaflet/app.js

```diff
@@ -0,0 +1,54 @@
+import * as Leaflet from 'leaflet';
```

Comment:
It seems canonical to import leaflet as `L` (or at least that's how it's exposed when included as a script, I think); don't have a strong preference whether we follow that.